src/en/mangademon/src/eu/kanade/tachiyomi/extension/en/mangademon/MangaDemon.kt

```diff
@@ -0,0 +1,137 @@
+package eu.kanade.tachiyomi.extension.en.mangademon
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.POST
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import okhttp3.Headers
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class MangaDemon : ParsedHttpSource() {
+
+    override val lang = "en"
+    override val supportsLatest = true
+    override val name = "MangaDemon"
```

Comment:
The site names itself "Manga Demon", so maybe follow it?

```suggestion
    override val name = "Manga Demon"
```